docs/reference/SourceDocsDemo/classes/ACLTestClass.md

```diff
@@ -31,3 +31,35 @@ public func publicMethod()
 ```
 
 > Method with `public` access level
+
+### `implicitInternalMethod()`
+
+```swift
+func implicitInternalMethod()
+```
+
+> Method with implicit `internal` access level
+
+### `explicitInternalMethod()`
+
+```swift
+internal func explicitInternalMethod()
+```
+
+> Method with explicit `internal` access level
+
+### `filePrivateMethod()`
+
+```swift
+fileprivate func filePrivateMethod()
+```
+
+> Method with explicit `fileprivate` access level
+
+### `privateMethod()`
+
+```swift
+private func privateMethod()
+```
+
+> Method with explicit `private` access level
```

Comment:
Should these methods be be generating documentation since they're `private` and `fileprivate`? Or is there a test change I'm missing somewhere?

Reply:
Yep, inside `Makefile`, for `SourceDocsDemo` I'm passing `--min-acl private`.